index.test-d.ts

```diff
@@ -76,7 +77,7 @@ export async function verifyRequestTest(
 
 export async function fetchVerificationKeysTest() {
   const result = await fetchVerificationKeys();
-  expectType<{ cacheId: string; keys: VerificationPublicKey[] }>(result);
+  expectType<VerificationKeysCache>(result);
```

Comment:
for tests, I'd leave the explicit assertions. This will help us to catch unintentional API changes. Say we accidentally remove or change a key in the `VerificationKeysCache` type. This test would still pass. But having it being explicit will fail, helping us prevent a regression.

It's also a good reminder to add additions to changelogs, as each addition needs to be added to the assertion